vnext/Microsoft.ReactNative/ReactInstanceSettings.idl

```diff
@@ -12,129 +12,199 @@ import "RedBoxHandler.idl";
+    DOC_STRING("Name of the JavaScript bundle file. If @.JavaScriptBundleFile is specified it is used instead.")
     String JavaScriptMainModuleName { get; set; };
 
-    DOC_STRING("The name of the JavaScript bundle file to load. This should be a relative path from @.BundleRootPath.  `.bundle` will be appended to the end, when looking for the bundle file.")
+    DOC_STRING(
+      "The name of the JavaScript bundle file to load. This should be a relative path from @.BundleRootPath. "
+      "The `.bundle` extension will be appended to the end, when looking for the bundle file.")
     DOC_DEFAULT("index.windows")
     String JavaScriptBundleFile { get; set; };
 
-    DOC_STRING("Should the instance run in a remote environment such as within a browser.\n\
-By default, this is using a browser navigated to  http://localhost:8081/debugger-ui served by Metro/Haul.\n\
-Debugging will start as soon as the react native instance is loaded.")
+    DOC_STRING(
+      "Should the instance run in a remote environment such as within a browser.\n"
+      "By default, this is using a browser navigated to http://localhost:8081/debugger-ui served by Metro/Haul.\n"
+      "Debugging will start as soon as the react native instance is loaded.")
     Boolean UseWebDebugger { get; set; };
 
-    DOC_STRING("Should the instance trigger the hot module reload logic when it first loads the instance.\n\
-Most edits should be visible within a second or two without the instance having to reload.\n\
-Non-compatible changes still cause full reloads.\n\
-See [Fast Refresh](https://reactnative.dev/docs/fast-refresh) for more information on Fast Refresh.")
+    DOC_STRING(
+      "Should the instance trigger the hot module reload logic when it first loads the instance.\n"
+      "Most edits should be visible within a second or two without the instance having to reload.\n"
+      "Non-compatible changes still cause full reloads.\n"
+      "See [Fast Refresh](https://reactnative.dev/docs/fast-refresh) for more information on Fast Refresh.")
     Boolean UseFastRefresh { get; set; };
 
-    DOC_STRING("Enable live reload to load the source bundle from the React Native packager.\n\
-When the file is saved, the packager will trigger reloading.\n\
-**For general use this has been replaced by @.UseFastRefresh.**")
+    DOC_STRING(
+      "Enable live reload to load the source bundle from the React Native packager.\n"
+      "When the file is saved, the packager will trigger reloading.\n"
+      "**For general use this has been replaced by @.UseFastRefresh.**")
     Boolean UseLiveReload { get; set; };
 
-    DOC_STRING("Enables debugging in the JavaScript engine (if supported).  \n\
-For Chakra this enables you to debug the JS runtime directly within your app using Visual Studio -> Attach to process (Script)")
+    DOC_STRING(
+      "Enables debugging in the JavaScript engine (if supported).\n"
```

Comment:
it would be useful to understand more of the "if supported" piece (e.g. point to docs about Hermes/...)

Reply:
I would leave to someone else. I do not know much about the debugging in Hermes.